library/constant_time_internal.h

```diff
@@ -129,6 +129,23 @@ unsigned mbedtls_ct_size_bool_eq( size_t x,
 unsigned mbedtls_ct_mpi_uint_lt( const mbedtls_mpi_uint x,
                                  const mbedtls_mpi_uint y );
 
+/**
+ * \brief          Check if one unsigned MPI is less than another in constant
+ *                 time.
+ *
+ * \param A        The left-hand MPI. This must point to an array of limbs
+ *                 with the same allocated length as \p B.
+ * \param B        The right-hand MPI. This must point to an array of limbs
+ *                 with the same allocated length as \p A.
+ * \param limbs     The number of limbs in \p A and \p B.
```

Comment:
This line doesn't line up with the rest
```suggestion
 * \param limbs    The number of limbs in \p A and \p B.
```